### PR TITLE
Supporting content

### PR DESCRIPTION
**Pull request by BjornTwachtmann**
HTML/CSS/Presenters for supporting content. See PROGRAMMES-5752.

### CODE REVIEW

File: app/Resources/views/find_by_pid/tlec.html.twig [hunk 1]
```diff
@@ -137,7 +137,15 @@
                     {%- endfor -%}
                 {%- endif -%}
             </ul>
-            {# TODO Supporting Content blocks #}
+            {%- if supportingContentItems -%}
+                <ul class="grid">
+                    {%- for supportingContentItem in supportingContentItems -%}
+                        <li class="1/1 1/2@gel4 grid__item">
```
Comment (BPScott) [inline app/Resources/views/find_by_pid/tlec.html.twig]: Don't need `1/1` as `grid_item` is width:100% by default

File: src/DsAmen/Organism/SupportingContent/_supporting-content.scss [hunk 2]
```diff
@@ -0,0 +1,26 @@
+.feature {
```
Comment (BPScott) [inline src/DsAmen/Organism/SupportingContent/_supporting-content.scss]: The CSS block should should match the filename, it's confusing when the names don't line up

File: src/DsAmen/Organism/SupportingContent/supporting_content.html.twig [hunk 3]
```diff
@@ -0,0 +1,21 @@
+{%- set grid_classes = "feature feature--noimage" -%}
+{%- if supporting_content.showImage() -%}
+    {%- set grid_classes = "feature grid" -%}
+{%- endif -%}
+<div class="br-box-secondary br-secondary-link">
+    <div class="{{ grid_classes }}">
+        {%- if supporting_content.showImage() -%}
+            <div class="grid__item feature__item 1/1 1/2@gel2b br-box-subtle">
```
Comment (BPScott) [inline src/DsAmen/Organism/SupportingContent/supporting_content.html.twig]: If we use flexbox for positioning the text and  image (configuring the text container to take up as much space as is available), then we could get rid of having to use the grid for this layout, which would save on having to do ths conditional logic around the grid__item classes.

Comment (BjornTwachtmann) [reply src/DsAmen/Organism/SupportingContent/supporting_content.html.twig]: You got an example? That media object seems to be where sanity goes to die.

**Pull request merged**